expenses-manager-frontend/src/main/frontend/src/app/edit-expense-dialog/edit-expense-dialog.component.html

```diff
@@ -0,0 +1,60 @@
+<h1><fa-icon [icon]="['fas', 'money-bill']"></fa-icon> Edit expense</h1>
```

Comment:
Aici m-as folosi tot de modalul de add-expense pentru a nu avea cod dublicat. Putem face asta prin transmiterea de date de la row-ul pe care s-a dat click catre modal, iar in modal verificam daca s-au trimit date sau nu, sa stim daca facem prefill la field-uri sau nu. De asemenea si titlul modalului se va modifica ca atare daca suntem in modul de Add sau Update, la fel si pe butonul de actiune.